source/_integrations/unifiprotect.markdown

```diff
@@ -0,0 +1,88 @@
+
+Example: as of `2022.2.0` of Home Assistant, UniFi Protect `1.21.0-beta.1` is the newest UniFi Protect version. So the recommended versions of UniFi Protect to run for a `2022.2.0` version of Home Assistant are `1.20.0`, `1.20.1` or `1.21.0-beta.1`.
+
+## Prerequisites
+
+Before you install this Integration you need to ensure that the following two settings are applied in UniFi Protect.
+
+### Local User
+
+1. Login to your *Local Portal* on your UniFiOS device, and click on *Users*
+1. In the upper right corner, click on *Add User*
+1. Click *Add Admin*, and fill out the form. Specific Fields to pay attention to:
+    * Role: Must be *Limited Admin*
+    * Account Type: *Local Access Only*
+    * CONTROLLER PERMISSIONS - Under UniFi Protect, select Administrators. **NOTE**: Other roles may work, but only the default Administrators rule is fully tested.
+1. Click *Add* in at the bottom Right.
+
+**HINT**: A few users have reported that they had to restart their UDMP device after creating the local user for it to work. So if you get some kind of *Error 500* when setting up the Integration, try restart the UDMP.
+
+![ADMIN_UNIFIOS](/images/integrations/unifiprotect/unifi_os_admin.png)
+
+### Camera Streams
+
+The Integration uses the RTSP(S) Streams as the Live Feed source, so this needs to be enabled on each camera. With the latest versions of UniFi Protect, the stream is enabled per default, but it is recommended to just check that this is done. To check and enable the the feature
+
+1. open UniFi Protect and click on *Devices*
+2. Select *Manage* in the Menu bar at the top
+3. Click on the + Sign next to RTSP
+4. Enable minimum 1 stream out of the 3 available. UniFi Protect will select the Stream with the Highest resolution as the default enabled one.
+
+#### Minimizing Latency
+
+To minimize the latency to your UniFi Protect cameras, you will want to [enable LL-HLS in the stream integration](/integrations/stream/#ll-hls). You will also want to put an HTTP/2 reserve proxy in front of Home Assistant so you can have connection pooling. If you do not add a reverse proxy, you may start to get "Waiting for Websocket..." messages while trying to view too many camera streams at once.
+
+##### Resources
+
+* NGINX Proxy Home Assistant OS Add-on: [![NGINX Proxy Add-on](https://my.home-assistant.io/badges/supervisor_addon.svg)](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_nginx_proxy)
+
+* [Community Guide for NGINX](https://community.home-assistant.io/t/reverse-proxy-using-nginx/196954)
```

Comment:
This should not be listed.

Reply:
This gets asked _a lot_ by people using the current HACS integration. "Why are do my cameras have a 15 second delay?"

Reply:
We do not support or recommend community guides; thus should not be listed as such.

Reply:
Oh, so just remove the community guide. That I can do. I can just change it to something like "if you are not using HA OS, look up a guide".

Reply:
Why not explain what it needs instead of saying "look up a guide"?

Reply:
I mean, I mention you need a HTTP/2 reverse proxy, but I am trying to avoid this being an official guide on how to do that. I mentioned to @allenporter it would be nice if we had an official one since using LL-HLS _strong_ recommends having HTTP/2, but I think the docs for this integration it is out of scope.

Reply:
>  but I am trying to avoid this being an official guide on how to do that. 

Correct. So don't do that.

Reply:
Cleaned up the wording and only reference the NGINX add-on making it less guide-like.